### ISSUE TITLE
[Import Connectors (likely more)] Define a Volume for any r/w filesystem locations

### ISSUE THREAD

**Issue by MaxwellDPS** [OpenCTI-Platform/connectors#2819]
## Description

In the import document connector the location used by [`_download_import_file()`](https://github.com/OpenCTI-Platform/connectors/blob/master/internal-import-file/import-document/src/reportimporter/core.py#L243) needs to be defined as a volume.

This poses a security issue if users decide to just not run with the root filesystem of a container due to having no context of this and turning it off.

More broadly k8s covers security contexts on runtime really well, I would recommend all containers be able to run non-root with a read only filesystem

Per Docker best practice is to make anyplace files are created at run time a volume.  
> You should use the VOLUME instruction to expose any database storage area, configuration storage, or **files and folders created by your Docker container**. You are strongly encouraged to use VOLUME for any combination of mutable or user-serviceable parts of your image.

## Environment

1. OS (where OpenCTI server runs): N/A
2. OpenCTI version: All so far
3. OpenCTI client: N/A
4. Other environment details: N/A

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. deploy import connector with a readonly root filesystem

## Expected Output

Proper volume definitions

## Actual Output

Runtime errors if no volume is mounted

## Additional information

https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
https://docs.docker.com/build/building/best-practices/#volume

**Comment by MaxwellDPS** [comment]
Same issue on threat fox, it isnt even using a directory and is writing to yield them. This would be easy to swap out for StringIO

https://github.com/OpenCTI-Platform/connectors/blob/master/external-import/threatfox/src/main.py#L301


This makes no sense 
```python
        try:
            zipped_file = io.BytesIO(data)
            with zipfile.ZipFile(zipped_file, "r") as zip_ref:
                with zip_ref.open("full.csv") as full_file:
                    csv_data = full_file.read()
        except zipfile.BadZipFile:
            # Treat as an unzipped CSV from /recent/
            csv_data = data

        with open(CSV_PATH, "wb") as fd:
            fd.write(csv_data)

        with open(CSV_PATH, "r", encoding="utf-8") as fd:
            yield from (line for line in fd if not line.startswith("#"))
```

Recommending this is changed to not use a generator as no memory saving is happening anyhow by using `csv_data = full_file.read()`
```python
        try:
            zipped_file = io.BytesIO(data)
            with zipfile.ZipFile(zipped_file, "r") as zip_ref:
                with zip_ref.open("full.csv") as full_file:
                    csv_data = full_file.read()
        except zipfile.BadZipFile:
            # Treat as an unzipped CSV from /recent/
            csv_data = data

        return [
            line 
            for line in csv_data.split() 
            if not line.startswith("#")
       ]
```

**Comment by richard-julien** [comment]
Yes @romain-filigran, i think @MaxwellDPS is right, there is no need to write the file in some connectors. That could be improved
